discord-canary.spec

```diff
@@ -29,15 +29,14 @@ mkdir -p $RPM_BUILD_ROOT/usr/share/applications
 
 cp -r * $RPM_BUILD_ROOT/opt/DiscordCanary/
 ln -sf /opt/DiscordCanary/DiscordCanary $RPM_BUILD_ROOT/%{_bindir}/
-install -m 755 Discord.desktop $RPM_BUILD_ROOT/usr/share/applications/
+desktop-file-validate %{buildroot}/%{_datadir}/applications/Discord.desktop
```

Comment:
I think I know why you got the error - it looks like rpmbuild will look for the .desktop file in the tgz -- and what I probably did when I last built this is modify the tgz to add the desktop file into it.